Makefile

```diff
@@ -81,10 +87,30 @@ test/e2e: test/e2e/go test/e2e/ansible test/e2e/helm
 test/e2e/go:
 	./hack/tests/e2e-go.sh $(ARGS)
 
-test/e2e/ansible:
+test/e2e/ansible: docker/build/ansible
 	./hack/tests/e2e-ansible.sh
 
-test/e2e/helm:
+test/e2e/helm: docker/build/helm
 	./hack/tests/e2e-helm.sh
 
 .PHONY: test test/sanity test/unit test/subcommand test/e2e test/e2e/go test/e2e/ansible test/e2e/helm test/ci-go test/ci-ansible
+
+docker: docker/build docker/push
```

Comment:
This recipe should be something more generic like `image:` since we'll be supporting `buildah` eventually.